script.js

```diff
@@ -1,4 +1,280 @@
+  var playerObjectIcon = getObjectIcon(playerObject);
+  var computerObjectIcon = getObjectIcon(computerObject);
+  return (
+    'The computer chose ' +
+    computerObject +
+    computerObjectIcon +
+    '<br><br>' +
+    'You chose ' +
+    playerObject +
+    playerObjectIcon
+  );
+};
+
+ // Project Part 2 - Choose normal or Reverse Mode
+let playRPS2 = function (input) {
+   // var myOutputValue = '';  
+  if (currentMode == 'waiting for user name') { 
+  
+    // set the name    
+    userName = input;   
+    currentMode = 'waiting for game type'; 
+    console.log(currentMode)
+    return 'Hello ' + userName + ` choose your type -- normal or reverse`;
+  }
+  // for current mode = normal, 
+  if (currentMode == 'waiting for game type') {
+    gameType = input
+    currentMode = 'game mode'; 
+    console.log('setting gameType')
+    return `Hello ${userName} you have chosen ${gameType} game type`;
+  }
+
+  if (currentMode === 'game mode') {
+    if (gameType === 'normal') {
+        let game = normalRPS(input)
+        return game
+    } else {
+        let game = reverseRPS(input)
+        return game
+    }
+  }
+};
+  
+let normalRPS = function(playerSign) {
+  let playerObject = playerSign
+  let computerObject = getComputerSign()
+  countGamesPlayed = countGamesPlayed +1 ;
+
+  // Get a default output message sharing what player's and computer's objects are
+  let defaultObjectsMessage = getDefaultObjectsMessage(
+    playerObject,
+    computerObject
+  );
+
+  // Compare player's object with computer's object and output win status
+  // a) draw case
+  if (doesPlayerDrawComputer (playerObject, computerObject) ){
+    countComputerWins = countComputerWins;
+    countGamesDraw = countGamesDraw + 1;
+    myOutputValue = defaultObjectsMessage + "<br><br> It's a draw! <br><br>  " +
+                    replayMessage + 
+                    `${countGamesPlayed} <br><br> Total Computer Wins is 
+                    ${countComputerWins}. <br><br> Total Draws is 
+                    ${countGamesDraw}.`
+  }
+
+  // b) player wins case
+  else if (doesPlayerBeatsComputer(playerObject, computerObject) ) {
+    countComputerWins = countComputerWins;
+    countGamesDraw = countGamesDraw
+    myOutputValue = defaultObjectsMessage + "<br><br> You win! <br><br> " + 
+                    replayMessage  + 
+                    `${countGamesPlayed} <br><br> Total Computer Wins is 
+                    ${countComputerWins}. <br><br> Total Draws is 
+                    ${countGamesDraw}.`
+  }
+  // c) computer wins case
+  else {
+    countComputerWins = countComputerWins +1;
+    countGamesDraw = countGamesDraw
+    myOutputValue = defaultObjectsMessage + "<br><br> You lose! Bummer. <br><br>  " + 
+                    replayMessage  + 
+                    `${countGamesPlayed} <br><br> Total Computer Wins is 
+                    ${countComputerWins}. <br><br> Total Draws is 
+                    ${countGamesDraw}.`
```

Comment:
Nice use of defaultObjectMessage for all win, loss, draw cases. As the bottom of the strings are identical for all three cases, you can attempt to make this a result returned by a function. Just like your getDefaultObjectsMessage function